### cp8_17 예외 되던지기

- 한 메서드에서 발생할 수 있는 예외가 여럿인경우

#### 예시

- 코드

        package cp8.ex8_12;

        public class Ex8_11 {

            public static void main(String[] args) {
                try {
                    method1();
                }
                catch(Exception e) {
                    System.out.println("main 메서드에서 예외가 처리되었습니다.");
                }
            }

            static void method1() throws Exception{
                try {
                    throw new Exception();
                }
                catch(Exception e) {
                    System.out.println("method1 메서드에서 예외가 처리되었습니다.");
                    throw e;
                }
            }
        }

- 결과

        method1 메서드에서 예외가 처리되었습니다.
        main 메서드에서 예외가 처리되었습니다.
